### ISSUE TITLE
Suspiciousness is either 0 or 1

### ISSUE THREAD

**Issue by JoranHonig** [JoranHonig/tarantula#2]
@cgewecke adding an issue here so we can figure out wha't going wrong. Continueing from https://github.com/sc-forks/solidity-coverage/issues/430#issuecomment-759162035

In short:
@cgewecke ran fault localisation on the coverage output for sushiswap, and got the following output:
 https://gist.github.com/cgewecke/2859f6e35aeb780f107757a45fd174c8https://gist.github.com/cgewecke/2859f6e35aeb780f107757a45fd174c8
 
 It's weird because there is no gradient in the score, only min or max suspiciousness. Maybe there is a bug in the library that causes this problem.

**Comment by JoranHonig** [comment]
@cgewecke can you share the mocha & coverage json files that you used as an input?

**Comment by cgewecke** [comment]
Yes, gists here:
+ [sushiswap.testMatrix.json][1]
+ [sushiswap.mochaOutput.json][3]
+ [sushiswap.coverage.json][2]

And my tarantula script was:

```js
const fl = require('tarantula-fl');
const mochaOutput = require('./mochaOutput.json');
const testMatrix = require('./testMatrix.json');

const testData = {
  testResults: fl.fromMocha(mochaOutput),
  coverage: fl.fromSolCover(testMatrix)
}

const score = fl.tarantulaScore(testData)
console.log(JSON.stringify(score, null, ' '));
```
[1]: https://gist.github.com/cgewecke/fec8e40070233599e132d9ac80f637f8
[2]: https://gist.github.com/cgewecke/366a7fe8f6bfc455220a4747b14b6877
[3]: https://gist.github.com/cgewecke/9fad5003eb83ff98032a01a20d9a8872

**Comment by cgewecke** [comment]
Also, I published that branch as a beta today. It's installable as
```
npm install --save-dev solidity-coverage@beta
```

The mocha JSON is automatically generated with the `--matrix` flag and written to `mochaOutput.json`

**Comment by JoranHonig** [comment]
@cgewecke I found the problem 👌 

I had no special treatment for situations where there were no failing tests. Should be fixed with v0.0.7!
